README.md

```diff
@@ -26,57 +28,47 @@ If this repo contains any reference architectures, uncomment the heading below a
 See "Reference architecture" in Authoring Guidelines in the public documentation at
 https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
 -->
-<!-- ## Reference architectures -->
 
+## Reference architectures
+
+[Maximo Application Suite Deployable Architecture](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mas/main/reference-architecture/mas_deployable_architecure.svg)
```

Comment:
Are you trying to show the diagram here? If so, I think you need to change this to:
```suggestion
[![Maximo Application Suite Deployable Architecture](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mas/main/reference-architecture/mas_deployable_architecure.svg)]
```